app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `Person` class is not only responsible for maintaining the state of a person but also for storing all people. This is a violation of the Single Responsibility Principle. A class should have only one responsibility. I suggest you create a separate class or function to handle the storage of all people.